ui/components/app/snaps/snap-delineator/snap-delineator.js

```diff
@@ -80,5 +80,5 @@ export const SnapDelineator = ({
 SnapDelineator.propTypes = {
   snapName: PropTypes.string,
   type: PropTypes.string,
-  children: PropTypes.ReactNode,
+  children: PropTypes.element,
```

Comment:
I'm not sure I understand this change. Why is `ReactNode` not enough? The issue linked mentions `children` to be undefined, that should never be the case 🤔

Reply:
PropTypes.ReactNode isn't a valid type, and you're right it should be node instead of element.